language/move-stdlib/modules/Signer.move

```diff
@@ -1,10 +1,10 @@
 address 0x1 {
 module Signer {
     // Borrows the address of the signer
-    // Conceptually, you can think of the `signer` as being a resource struct wrapper arround an
+    // Conceptually, you can think of the `signer` as being a struct wrapper arround an
     // address
     // ```
-    // struct Signer has key, store { addr: address }
+    // struct Signer has copy, drop { addr: address }
```

Comment:
Signer doesn't have copy, so this should just be drop

Reply:
I see. I just fixed it.